_drafts/SC3-3-2-form-field-has-name.md

```diff
@@ -0,0 +1,77 @@
+---
+name: Form field has accessible name
+description: |
+  Each form field element has an accessible name
+
+success_criterion:
+- 3.3.2 # Labels or Instructions
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to:
+- any element that is [exposed to assistive technologies](#exposed-to-assistive-technologies), and
+- that has a [semantic role](#semantic-role) that inherits from the [abstract](https://www.w3.org/TR/wai-aria/#abstract_roles) `input` or `select` role, and 
+- does not have a [required context](https://www.w3.org/TR/wai-aria/#scope) role that itself inherits from one of those roles.
```

Comment:
I don't even know what that means. This clearly doesn't pass the "plain language" requirement - just give people a list of roles to look for.

Reply:
Last point could be changed to:

> - does not have a [required context](https://www.w3.org/TR/wai-aria/#scope) role that itself inherits from the `input` or `select` role.

Does that help?

Reply:
I think point 2 and 3 can be joined to one.

- that has a [semantic role](#semantic-role) that inherits from the [abstract](https://www.w3.org/TR/wai-aria/#abstract_roles) `input` or `select` role, and does not have a [required context](https://www.w3.org/TR/wai-aria/#scope) role.